Project Rogue/Assets/Scripts/StateSystem/StateHandler.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System;
+
+public class StateHandler : MonoBehaviour
+{
+#region STATES
+    [Flags]
+    public enum State : byte
+    {
+        Active = 1 << 0,
```

Comment:
These are jump temp states, we can them to what ever, we could even add multiple types of states in later